examples/hubble-cli.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hubble-cli
+  labels:
+    app.kubernetes.io/name: hubble-cli
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: hubble-cli
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: hubble-cli
+    spec:
+      containers:
+      - name: hubble-cli
+        image: quay.io/cilium/hubble:v0.8.0
+        imagePullPolicy: Always
```

Comment:
`IfNotPresent`

Reply:
Also updated in v1.10 / master through https://github.com/cilium/cilium/pull/16537